examples/handlebars/components/render/render.spec.js

```diff
@@ -0,0 +1,32 @@
+const fractal = require('../../fractal.config.js');
+
+describe('render', () => {
+    beforeEach(async () => {
+        await fractal.load();
+    });
+
+    it('renders default template for default variant', async () => {
+        const render = await fractal.components.find('@render').render();
+        expect(render).toBe('Render something\n');
+    });
+
+    it('renders default template for regular variant', async () => {
+        const render = await fractal.components.find('@render--variant-1').render();
+        expect(render).toBe('Render something\n');
```

Comment:
I think we should change the content of this variant to make it unique and thus prevent any future bug where the template used would be wrong but the test would still pass.

Reply:
Agreed!